dev/com.ibm.ws.security.acme/resources/com/ibm/ws/security/acme/resources/AcmeMessages.nlsprops

```diff
@@ -10,19 +10,123 @@
 ###############################################################################
 # -------------------------------------------------------------------------------------------------
 #CMVCPATHNAME com.ibm.ws.security.acme/resources/com/ibm/ws/security/acme/resources/AcmeMessages.nlsprops
-#COMPONENTPREFIX CWWKX
+#COMPONENTPREFIX CWPKI
 #COMPONENTNAMEFOR WebSphere ACME
 #ISMESSAGEFILE TRUE
 #NLS_MESSAGEFORMAT_VAR
 #NLS_ENCODING=UNICODE
 # -------------------------------------------------------------------------------------------------
+# Message prefix block: CWPKI2000 - CWPKI2299
+#-----  ACME messages
 
-# TODO: Get the correct Message Prefix block for ACME feature
-# liberty wiki should have a list of the Message Prefix
-# Message prefix block: CWWKJ0200 - CWWKJ0299
+# The ACME referred to in these messages stands for Automatic Certificate Management Environment.
+# The term "certificate authority" is in reference to SSL/TLS certificate providers.
+# The URI parameter in these messages is for the ACME certificate authority, such as https://sampleCAProvider.org/directory
 
-#-----  ACME messages
+CWPKI2001E=CWPKI2001E: The ACME certificate authority at the {0} URI responded that the authorization challenge failed for the {1} domain. The challenge status is {2}.  The error is {3}.
+CWPKI2001E.explanation=The challenge status indicated that the authorization challenge request failed and a certificate cannot be created.
+CWPKI2001E.useraction=Review the status message and error for details on the failure.
+
+CWPKI2002E=CWPKI2002E: Polling for a successful authorization challenge for the {0} domain from the ACME certificate authority at the {1} URI timed out. The status is {2}. The configured timeout is {3}.
```

Comment:
**change**
Polling for a successful authorization challenge for the {0} domain from the ACME certificate authority at the {1} URI timed out.
**to**
Polling timed out for a successful authorization challenge for the {0} domain from the ACME certificate authority at the {1} URI.